docker-compose.yml

```diff
@@ -30,7 +30,13 @@ services:
     image: ghcr.io/sanctuuary/restape:latest
     restart: always
     ports:
-      - "4444:4444"
+      - "8080:4444"
+  
+  pubmetric:
+    image: ghcr.io/workflomics/pubmetric:pr-43
```

Comment:
@vedran-kasalica what about this one? Should it also point to latest?

Reply:
'pubmetric' was not merged yet into main yet, so we can either do it now or wait a week or two to do so.